### PR TITLE
Update ASP.NET Core known issues for .NET Core 3.0 Preview 4

### PR DESCRIPTION
**Pull request by danroth27**

### CODE REVIEW

**Comment by danroth27** [comment]
@mkArtakMSFT

File: release-notes/3.0/preview/3.0.0-preview-known-issues.md [hunk 1]
```diff
@@ -84,107 +46,37 @@ This document lists known issues for **.NET Core 3.0 Preview releases** which ma
...
-            </Layout>
-          );
-        }
-      }
+- **Server-side Blazor app fails to run on non-Windows platforms**
+
+    Running a server-side Blazor app on a non-Windows platform fails with the following error:
+
+    ```
+    System.InvalidOperationException: Could not load the embedded file manifest 'Microsoft.Extensions.FileProviders.Embedded.Manifest.xml' for assembly 'Microsoft.AspNetCore.Components.Server'.
```
Comment (pranavkm) [inline release-notes/3.0/preview/3.0.0-preview-known-issues.md]: Would you like me to investigate possible workarounds for this?

Comment (danroth27) [reply release-notes/3.0/preview/3.0.0-preview-known-issues.md]: That would be lovely

File: release-notes/3.0/preview/3.0.0-preview-known-issues.md [hunk 2]
```diff
@@ -2,46 +2,8 @@
 
 This document lists known issues for **.NET Core 3.0 Preview releases** which may be encountered during usage.
 
-
 ## ASP.NET Core
 
-### Preview 3
-- **The Windows Hosting bundle doesn't exist for Preview3**: The Windows Hosting bundle contained an issue which copied the 32 bit dlls into 64 bit locations. To work aroud this issue, please install the 3.0 Preview 2 [Hosting Bundle](https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-preview2-download.md) and run the executable on a command line with the paramaters `OPT_NO_SHAREDFX=1 OPT_NO_RUNTIME=1`. Then install the appropriate .NET and ASP.NET runtimes for preview3.
-
-- **Updates to .razor files fail to show up in subsequent builds**: Updates to Razor Component (.razor) in Visual Studio may fail to show up in subsequent builds. To work around this issue add the following item group to the project file:
```
Comment (vivmishra) [inline release-notes/3.0/preview/3.0.0-preview-known-issues.md]: Add header text like [RESOLVED]

Comment (danroth27) [reply release-notes/3.0/preview/3.0.0-preview-known-issues.md]: Done

**Pull request merged**